README.md

```diff
@@ -4,27 +4,68 @@
   [![NuGet](https://img.shields.io/nuget/dt/Soneta.Sdk.svg)](https://www.nuget.org/packages/Soneta.Sdk)
   [![Build Status](https://soneta.visualstudio.com/GitHub/_apis/build/status/Soneta.MsBuild.SDK?branchName=master)](https://soneta.visualstudio.com/GitHub/_build/latest?definitionId=2&branchName=master)
   
-## Wstęp 
-Sdk stworzone przez firmę Soneta pozwalające automatycznie skonfigurować oraz uzupełnić projekty dodatków o niezbędne elementy potrzebne do współpracy z oprogramowaniem enova.
+# Wstęp 
+SDK (Softwre Development Kit) jest to zestaw narzędzi dla programistów niezbędnych w tworzeniu aplikacji z danej biblioteki. Soneta SDK jest zestawem narzędzi niezbędnym do tworzenie dodatków dla sytemu enova365. Pozwala automatycznie skonfigurować projekt oraz uzupełniać projekty dodatku o niezbędne elementy potrzebne do wsprółpracy z oprogramowaniem enova365.Soneta.SDK jest projektem platformy .NET w którego skład wchodzą takie pliki jak:<br>
+<ul>
```

Comment:
Może coś takiego :
Do elementów konfiguracyjnych Soneta SDK zaliczamy następujące pliki wraz z ich przeznaczeniem:
Common.item.props
Plik zapewnia automatyczną obsługę dołączania nowych plików do projektu. Między innymi pliki *.pageform.xml, *dbinit.xml zostaną automatycznie skonfigurowane jako EmbeddResource dla własności Build Action.
Sdk.props
Plik zapewnia poprawne utworzenie referencji do bibliotek systemu enova365 i jego otoczenia dla różnych typów projektów np. Dla projektu warstwy testującej zostaną dołączone biblioteki testowe.
Skd.targets
Plik przechowuje informacje dotyczące procesu budowania tworzonego rozszerzenia enova365 m.in. automatycznego uruchamiania generatora tworzenia klas bazowych stanowiących definicje obiektów baz danych (*.business.cs) na podstawie przygotowanych opisów tabel w plikach xml (*.business.xml). Dzięki takiemu podejściu twórca rozszerzenia nie będzie musiał się zastanawiać, którą wersję generatora powinien zastosować(podpiąć) - zostanie to wykonane automatycznie.